config/ember-try.js

```diff
@@ -58,19 +74,6 @@ module.exports = async function () {
           devDependencies: {},
         },
       },
-      {
-        name: 'ember-default-with-jquery',
```

Comment:
not sure if we needed this scenario anymore, but we don't use jquery in the addon and it caused issues for the v4